boxes/boxes/vite/playwright.config.ts

```diff
@@ -5,7 +5,7 @@ export default defineConfig({
   testMatch: "**.spec.ts",
   fullyParallel: true,
   retries: 3,
-  workers: process.env.CI ? 1 : 3,
```

Comment:
Boxes are not very robust and when testing locally with 3 workers, they can run into a a race condition when crafting the deployment tx in parallel (all of them try to register the contract and the tx fails with a duplicate nullifier, making the test retry)